k8s/config/rbac/manager_account.yaml

```diff
@@ -71,5 +64,5 @@ roleRef:
   kind: ClusterRole
   name: scheduler-plugin-role
 subjects:
-- kind: ServiceAccount
-  name: manager
+  - kind: ServiceAccount
+    name: manager
```

Comment:
DON'T mix the format changes into pull requests where new features/new changes are made.